### ISSUE TITLE
Use case: display recipe details

### ISSUE THREAD

**Issue by 1101011** [COMP-380-6/recipes#9]
Depends on #7 

Search results only contain basic information about a recipe - enough to interest the user in the result. More information is necessary should the user decide they want to prepare it. Upon clicking the recipe (probably its name or image), a modal box should open, displaying the following:

* Name
* Image
* Description
* Price
* Preparation time
* Popularity
* Healthiness
* Serving size
* Preparation instructions
* Required ingredients

There should be a close button to close the modal box and return to the "main page" that contains their search results.

**Comment by 1101011** [comment]
Mostly done. Will open another issue with a narrower scope.
